advanced_alchemy/mixins/audit.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from datetime import datetime, timezone
+
+from sqlalchemy.orm import Mapped, declarative_mixin, mapped_column, validates
+
+from advanced_alchemy.types import DateTimeUTC
+
+
+@declarative_mixin
```

Comment:
why was this added for all mixins? AFAIK this is not needed in 2.0

Reply:
I added these to be consistent across all mixins.  We had one with them and the rest without.

We can remove them all of none of them are needed; but I'd just like it to be consistent.